core/templates/services/debouncer.service.ts

```diff
@@ -27,29 +27,30 @@ export class DebouncerService {
   // be invoked. The function only gets executed after it stops being called
   // for `wait` milliseconds.
   debounce(func: () => void, millisecsToWait: number): () => void {
-    let timeout;
-    let context = this;
-    let args = arguments;
-    let timestamp;
-    let result;
+    let timeout: ReturnType<typeof setTimeout>|null;
+    let context: this|null = this;
+    // Arguments are of no use in the function
+    // as they are not getting any value also
+    // so removed args = Arguments.
```

Comment:
Remove this comment. Next time create a comment explaining this on GitHub.

Reply:
Done